From: Jesse Luehrs <doy@tozt.net>
Date: Mon, 24 Mar 2014 10:30:49 -0400
Subject: allow the systemd service to also be stopped

---
 tpacpi.service | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tpacpi.service b/tpacpi.service
index f996a78..4446b70 100644
--- a/tpacpi.service
+++ b/tpacpi.service
@@ -3,8 +3,11 @@ Description=sets battery thresholds
 
 [Service]
 Type=oneshot
+RemainAfterExit=yes
 ExecStart=/usr/bin/tpacpi-bat -s ST 0 40
 ExecStart=/usr/bin/tpacpi-bat -s SP 0 80
+ExecStop=/usr/bin/tpacpi-bat -s ST 0 0
+ExecStop=/usr/bin/tpacpi-bat -s SP 0 0
 
 [Install]
 WantedBy=multi-user.target
