From: Piotr Morgwai Kotarbinski <github@morgwai.pl>
Date: Sat, 3 May 2014 15:27:33 +0700
Subject: sanitize the path before calling an external program

---
 tpacpi-bat | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tpacpi-bat b/tpacpi-bat
index 6d84123..ecacb35 100755
--- a/tpacpi-bat
+++ b/tpacpi-bat
@@ -387,6 +387,7 @@ sub acpiCall($){
   my $call = shift;
 
   if(not -e $acpiCallDev){
+    $ENV{'PATH'} = '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin';
     system "modprobe acpi_call";
   }
   if(not -e $acpiCallDev){
